### ISSUE TITLE
VRF hard crash if terminal unit list error

### ISSUE THREAD

**Issue by EnergyArchmage** [NREL/EnergyPlus#4952]
While developing VRF input files, found that the program can hard crash with no information about the problem, when the input for terminal unit list input is wrong.  Adding the following around line line 2110 in HVACVariableRefrigerantFlow.cc solved the problem.  a zero value for TUListIndex needs to be trapped sooner.  

```
            // can hard crash before last fatal if TUListIndex is zero
            if ( ErrorsFound ) {
                ShowFatalError( RoutineName + "Errors found in getting AirConditioner:VariableRefrigerantFlow system input. Preceding condition(s) causes termination." );
            }
```

**Comment by rraustad** [comment]
PR #5141

VRF unit test is already created for VRF defrost issue. I can create unit test when that issue is merged, or add to that unit test.

**Comment by rraustad** [comment]
Complete. See https://github.com/NREL/EnergyPlus/pull/5164

**Comment by mjwitte** [comment]
Closed via #5164
